### PR TITLE
add missing types and functions to fluxible 

### PR DESCRIPTION
**Pull request by xbIm**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/fluxible-addons-react/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Type definitions for fluxible-addons-react 0.2
+// Project: https://github.com/yahoo/fluxible#readme
+// Definitions by: xbim <https://github.com/me>
```
Comment (RyanCavanaugh) [inline types/fluxible-addons-react/index.d.ts]: Your github username is not `me`

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

**Comment by xbIm** [comment]
@RyanCavanaugh could you, please, take a look?

**Comment by xbIm** [comment]
up, could some please take a look?

File: types/fluxible-addons-react/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// Type definitions for fluxible-addons-react 0.2
```
Comment (andy-ms) [inline types/fluxible-addons-react/index.d.ts]: There should be a file `types/fluxible-addons-react/fluxible-addons-react-tests.ts` with an example usage of this package.

Comment (xbIm) [reply types/fluxible-addons-react/index.d.ts]: thx for notificing.
I've added the test file, pleaste take a look

Comment (andy-ms) [reply types/fluxible-addons-react/index.d.ts]: Thanks!

**Pull request merged**